config.json

```diff
@@ -461,7 +461,12 @@
       "core": false,
       "difficulty": 6,
       "slug": "queen-attack",
-      "topics": null,
+      "topics": [
+        "arrays",
+        "matrices",
+        "conditionals",
+        "mathematics"
```

Comment:
These look like great suggestions! :) I would also add `games` as a topic since this is to do with chess :) Do you agree?

Reply:
@FridaTveit 
games and also how about **tuples** ? Can we add even though its not java datatype? or I should specify as **classes** as this specific program contains custom class BoardCoordinate .

Reply:
I wouldn't add `tuples` since it's not a java datatype so it could be confusing. But `classes` is a very good suggestion! :)

Reply:
I have done the updates